### DISCUSSION TITLE
Working with disabled inputs.

### DISCUSSION THREAD

**Discussion by adriantrunzo** [react-hook-form/react-hook-form#2654] [Q&A] [answered]
I was surprised to find that the values for disabled, registered inputs still appear in the `handleSubmit` data. I could not find any information about working with disabled form fields in the documentation, so I thought I would ask here:

What is the recommend approach for working with disabled inputs? Should I be registering and unregistering the input when the disabled property changes? For clarification, I am thinking of the basic case of a disabled text input:

```
<input name="example" ref={register} disabled />
```

With the current behavior I still see `example` in the `handleSubmit` data and my goal is for that key to be omitted since the corresponding input is disabled.

Thanks for the help!

**Comment by bluebill1049** [comment, marked as answer]
This is a bug, please raise an issue with it.
